examples/swarm/anomaly_swarming_hotgym/swarm/description.py

```diff
@@ -0,0 +1,423 @@
+
+  # Input stream specification per py/nupic/cluster/database/StreamDef.json.
+  #
+  'dataset' : {
+        u'info': u'test_hotgym',
+        u'streams': [   {   u'columns': [u'*'],
+                            u'info': u'hotGym.csv',
+                            u'source': u'file://extra/hotgym/hotgym.csv'}],
+         'aggregation': config['aggregationInfo'],
+        u'version': 1},
+
+  # Iteration count: maximum number of iterations.  Each iteration corresponds
+  # to one record from the (possibly aggregated) dataset.  The task is
+  # terminated when either number of iterations reaches iterationCount or
+  # all records in the (possibly aggregated) database have been processed,
+  # whichever occurs first.
+  #
+  # iterationCount of -1 = iterate over the entire dataset
+  'iterationCount' : 6000,
+
+
+  # A dictionary containing all the supplementary parameters for inference
+  "inferenceArgs":{'predictedField': config['predictedField'],
+                   'predictionSteps': config['predictionSteps']},
+
+  # Metrics: A list of MetricSpecs that instantiate the metrics that are
+  # computed for this experiment
+  'metrics':[],
+
+  # Logged Metrics: A sequence of regular expressions that specify which of
+  # the metrics from the Inference Specifications section MUST be logged for
+  # every prediction. The regex's correspond to the automatically generated
+  # metric labels. This is similar to the way the optimization metric is
+  # specified in permutations.py.
+  'loggedMetrics': ['.*aae.*'],
+}
+
+# Add multi-step prediction metrics
+for steps in config['predictionSteps']:
+  control['metrics'].append(
+      MetricSpec(field=config['predictedField'], metric='multiStep',
+                 inferenceElement='multiStepBestPredictions',
+                 params={'errorMetric': 'aae', 'window': 1000, 'steps': steps}))
+  control['metrics'].append(
+      MetricSpec(field=config['predictedField'], metric='trivial',
+                 inferenceElement='prediction',
+                 params={'errorMetric': 'aae', 'window': 1000, 'steps': steps}))
+  control['metrics'].append(
+      MetricSpec(field=config['predictedField'], metric='multiStep',
+                 inferenceElement='multiStepBestPredictions',
+                 params={'errorMetric': 'altMAPE', 'window': 1000, 'steps': steps}))
+  control['metrics'].append(
+      MetricSpec(field=config['predictedField'], metric='trivial',
+                 inferenceElement='prediction',
+                 params={'errorMetric': 'altMAPE', 'window': 1000, 'steps': steps}))
+  control['metrics'].append(
+      MetricSpec(field=config['predictedField'], metric='anomaly',
+                 inferenceElement='multiStepBestPredictions',
+                 params={'errorMetric': 'altMAPE', 'window': 1000, 'steps': steps,
+                         'desiredPct': 0.1, 'modelName': config['name']}))
+
+################################################################################
```

Comment:
Please remove these smudged lines and keep a space of 3 line breaks after previous method.